src/core/InlineBlock.js

```diff
@@ -1,6 +1,6 @@
 import sqlTokenTypes from "./tokenTypes";
 
-const INLINE_MAX_LENGTH = 35;
+const INLINE_MAX_LENGTH = 50;
```

Comment:
Thats a good idea, we already have them: `formats simple INSERT` query and `breaks long parenthized lists to multiple lines` in `behavesLikeSqlFormatter.js`